detection_rules/devtools.py

```diff
@@ -147,6 +152,43 @@ def build_integration_docs(ctx: click.Context, registry_version: str, pre: str,
     return docs
 
 
+@dev_group.command("bump-versions")
+@click.option("--major", is_flag=True, help="bump the major version")
+@click.option("--minor", is_flag=True, help="bump the minor version")
+@click.option("--patch", is_flag=True, help="bump the patch version")
+@click.option("--save", is_flag=True, help="Update the setup.cfg and packages.yml file")
+def bump_versions(major, minor, patch, save):
+    """Bump the versions"""
+
+    package = load_etc_dump('packages.yml')['package']
+    ver = package["name"]
+    new_version = Version(ver).bump(major, minor, patch)
+
+    kibana_version = f"^{new_version}.0" if not patch else "^{new_version}"
```

Comment:
I think this line is supposed to be `kibana_version = f"^{new_version}.0" if not patch else f"^{new_version}"`
Otherwise this happens:

```
detection-rules on  create-setup-cfg is  v8.6 via  v3.8.10 (venv) on  eric.forte 
❯ python -m detection_rules dev bump-versions --patch

█▀▀▄ ▄▄▄ ▄▄▄ ▄▄▄ ▄▄▄ ▄▄▄ ▄▄▄ ▄▄▄ ▄   ▄      █▀▀▄ ▄  ▄ ▄   ▄▄▄ ▄▄▄
█  █ █▄▄  █  █▄▄ █    █   █  █ █ █▀▄ █      █▄▄▀ █  █ █   █▄▄ █▄▄
█▄▄▀ █▄▄  █  █▄▄ █▄▄  █  ▄█▄ █▄█ █ ▀▄█      █ ▀▄ █▄▄█ █▄▄ █▄▄ ▄▄█

New package version: 8.7.1
New registry data version: 8.7.1-dev.0
New Kibana version: ^{new_version}

```